src/lighteval/few_shot_manager.py

```diff
@@ -230,7 +233,12 @@ def fewshot_context(
 
         if use_chat_template:
             output = self.get_examples_with_chat_template(
-                task=task, tokenizer=tokenizer, example=example, instruction=instruction, fewshot_ex=fewshot_ex
+                task=task,
+                tokenizer=tokenizer,
+                example=example,
+                instruction=instruction,
+                fewshot_ex=fewshot_ex,
```

Comment:
Does this wrap the examples in user/assistant turns? I think that was the main thing Eleuther was struggling to agree on vs just putting all the examples in a single user turn